packages/flame/lib/src/game/camera.dart

```diff
@@ -89,29 +89,36 @@ class Camera {
   double cameraSpeed = defaultCameraSpeed;
   double shakeIntensity = defaultShakeIntensity;
 
-  /// This is the current position of the camera, ie the world coordinate that is
-  /// rendered on the top left of the screen (origin of the screen space).
+  /// This is the current position of the camera, ie the world coordinate that
+  /// is rendered on the top left of the screen (origin of the screen space).
   ///
   /// Zero means no translation is applied.
   /// You can't change this directly; the camera will handle all ongoing
   /// movements so they smoothly transition.
   /// If you want to immediately snap the camera to a new place, you can do:
   /// ```
-  ///   camera.moveTo(newPosition);
-  ///   camera.snap();
+  ///   camera.snapTo(newPosition);
   /// ```
-  Vector2 get position => _position.clone();
+  Vector2 get position => __position.clone();
 
-  final Vector2 _position = Vector2.zero();
+  /// Do not change this directly since it bypasses [onPositionUpdate]
+  final Vector2 __position = Vector2.zero();
```

Comment:
do we need _position and __position? that sounds a bit confusing

Reply:
ah I think we only have `__position` here so could it be a typo?

Reply:
It's not a typo and that was the problem I was talking about in discord. So `__position` is the underlying Vector2 for the `_position` getter and setter, do you have any better way of doing this? I do agree that it looks weird.

Reply:
> for the _position getter and setter
can't that be called position?

like we have all three?

__position
_position
position

or can we promote _position to position?

if we do have 3 I would prefer to give them each a unique name explaining the difference

Reply:
Fixed, could you have a look again?